README.md

```diff
@@ -3,7 +3,8 @@
 ## Coalesces corporate storage files into larger objects
 
 This repo contains Makefile, and Dockerfile to fit the standard pattern.
-This repo is a base to create new Docker image repos, adding the githooks submodule, making the repo ready for use.
+This repo is a base to create new Docker image repos, adding the githooks
+submodule, making the repo ready for use.
```

Comment:
can you update the repo to reflect what this codebase does, what it publishes as a release, how to configure it etc?